cmd/qri.go

```diff
@@ -32,6 +32,10 @@ https://github.com/qri-io/qri/issues`,
 			if err == nil && noColor {
 				setNoColor(noColor)
 			}
+			// if it's a tty terminal disable color
+			if ioStreams.IsTerminal() {
```

Comment:
I think this is having the opposite effect of what's intended? Do we want `IsCygwinTerminal` instead? I'm not an expert here

Reply:
Still not sure on either option based on this: https://github.com/qri-io/qri/pull/1262#pullrequestreview-389020225